skbio/io/format/_sequence_feature_vocabulary.py

```diff
@@ -0,0 +1,351 @@
+    e_right = r'(?P<ER><?\d+\.\.>?[a-zA-Z_0-9\.]+:\d+)'
+    illegal = r'(?P<ILLEGAL>.+)'
+    # The order of tokens in the master regular expression also
+    # matters. When matching, re tries to match pattens in the order
+    # specified. Thus, if a pattern happens to be a substring of a
+    # longer pattern, you need to make sure the longer pattern goes
+    # first.
+    master_pat = re.compile('|'.join(
+        operators + [WS, LPAREN, RPAREN, COMMA,
+                     b, c, d, e_left, e_right, a,
+                     illegal]))
+
+    scanner = master_pat.scanner(loc_str)
+
+    bounds = []
+    fuzzy = []
+
+    metadata = {'strand': '+'}
+
+    for m in iter(scanner.match, None):
+        p, v = m.lastgroup, m.group()
+        if v == 'complement':
+            metadata['strand'] = '-'
+        elif p == 'A':
+            start = int(v)
+            bounds.append((start-1, start))
+            fuzzy.append((False, False))
+        elif p == 'B':
+            start, end = v.split('^')
+            start = int(start)
+            bounds.append((start-1, start))
+            fuzzy.append((False, False))
+        elif p == 'C' or p == 'D':
+            if p == 'C':
+                start, end = v.split('.')
+            else:
+                start, end = v.split('..')
+            fuzzy_s = fuzzy_e = False
+            if start.startswith('<'):
+                start = start[1:]
+                fuzzy_s = True
+            if end.startswith('>'):
+                end = end[1:]
+                fuzzy_e = True
+            bounds.append((int(start)-1, int(end)))
+            fuzzy.append((fuzzy_s, fuzzy_e))
+        elif p == 'ILLEGAL':
+            raise FileFormatError(
+                'Could not parse location string: "%s"' % loc_str)
+
+    return bounds, fuzzy, metadata
+
+
+def _serialize_feature_table(intervals, indent=21):
+    '''
+    Parameters
+    ----------
+    intervals : list of ``Interval``
+    '''
+    for intvl in intervals:
+        yield _serialize_single_feature(intvl, indent)
+
+
+def _serialize_single_feature(intvl, indent=21):
+    '''
+    Parameters
+    ----------
+    intvl : Interval
+    '''
+    # there are 5 spaces before Feature Key starts.
+    padding = ' ' * 5
+    qualifiers = []
+    md = intvl.metadata
+    voca_skip = _vocabulary_skip('insdc')
+    voca_change = _vocabulary_change('insdc', read_in=False)
+    # sort it so the output order is deterministic
+    for k in sorted(md):
+        if k.startswith('__') or k in voca_skip:
+            continue
+        v = md[k]
+        if k == 'phase':
+            v = str(v + 1)
+        if k in voca_change:
+            k = voca_change[k]
+        if isinstance(v, list):
+            for vi in v:
+                qualifiers.append(_serialize_qualifier(k, vi))
+        else:
+            qualifiers.append(_serialize_qualifier(k, v))
+
+    if '__location' in md:
+        loc = md['__location']
```

Comment:
It is out of our control, but any key starting with "___" is private and should be modified with care. Add explanation in line 139.